modules/local/openms_textexporter.nf

```diff
@@ -2,13 +2,13 @@ process OPENMS_TEXTEXPORTER {
     tag "$meta.id"
     label 'process_single'
 
-    conda "bioconda::openms=2.9.1"
+    conda "bioconda::openms=3.0.0"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/openms:2.9.1--h135471a_1' :
-        'quay.io/biocontainers/openms:2.9.1--h135471a_1' }"
+        'https://depot.galaxyproject.org/singularity/openms:3.0.0--h8964181_1' :
+        'quay.io/biocontainers/openms:3.0.0--h8964181_1' }"
 
     input:
-        tuple val(meta), path(consensus_resolved)
+        tuple val(meta), path(file)
```

Comment:
Why are there descriptive names in the other modules and here file?

Reply:
Because the textexport can take different formates, e.g. idXML, consensusXML etc.